### PR TITLE
added display-type to generated serialized json for attributes

### PR DESCRIPTION
**Pull request by bjartek**
## Description

The generated json does not include display_type

Sorry for whitespace mod here. lmk if you want me to redo.

### CODE REVIEW

File: cadence/contracts/utils/SerializeMetadata.cdc [hunk 1]
```diff
@@ -129,9 +129,10 @@ access(all) contract SerializeMetadata {
                 continue
             }
             serializedResult = serializedResult.concat("{")
-                .concat("\"trait_type\": ").concat(Serialize.tryToJSONString(trait.name)!)
-                .concat(", \"value\": ").concat(value!)
-                .concat("}")
+            .concat("\"trait_type\": ").concat(Serialize.tryToJSONString(trait.name)!)
+            .concat(", \"display_type\": ").concat(Serialize.tryToJSONString(trait.display_type)!)
```
Comment (sisyphusSmiling) [inline cadence/contracts/utils/SerializeMetadata.cdc:133]:
```suggestion
            .concat(", \"display_type\": ").concat(Serialize.tryToJSONString(trait.displayType)!)
```

**Review by sisyphusSmiling — CHANGES_REQUESTED** [review]
Thanks for the PR, and good call including the `display_type` value. I do wonder if we should only include a serialized `trait.displayType` if it's not `nil`.

**Comment by bjartek** [comment]
That is probably wise yes

**Comment by sisyphusSmiling** [comment]
@bjartek in the interest of moving the PR along and make the suggested changes, I'll update to target a feature branch and pick up the PR

**Pull request merged**